sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -49,11 +50,11 @@ src_prepare() {
 
 	# bug 598647 - PIE not supported
 	sed -e '/^CFLAGS/s:$: -fno-PIE:' -i Make.common || die
-
+	LIB_DIR="$(get_libdir)" || die
 	# Prepare UDK workspace
 	if ! use gnuefi; then
 		mkdir "${UDK_WORKSPACE}" || die
-		ln -s "${EPREFIX}/usr/lib/udk/"{Mde,IntelFramework}{,Module}Pkg \
+		ln -s "${EPREFIX}${LIB_DIR}/udk/"{Mde,IntelFramework}{,Module}Pkg \
```

Comment:
Have you tried compiling rEFInd without the gnuefi flag (this would require a change in Gentoo profile)? Because I'm not sure that UDK installation is already done in the correct library…

Reply:
I can try it the next time I have some time to work on my laptop.

Reply:
I will try this now